com.unity.render-pipelines.universal/Documentation~/2DRendererData_overview.md

```diff
@@ -32,3 +32,9 @@ All layers captured for use in the supplied texture will be drawn from the very
 
 ### Downsampling Method
 Downsampling will reduce the texture resolution used by _CameraSortingLayerTexture_. The options are: __None__, __2x Bilinear__, __4x Box__, __4x Bilinear__
+
+## Renderer Features
+
+The 2D Renderer supports Renderer Features. The setup for the features are called before any of the 2D built-in passes are queued.
```

Comment:
Give a brief explanation about what 'Render Features' are and maybe what they are for in this line.

Reply:
I think this is addressed by the line below (link to renderer feature doc page)?

Reply:
The user would need to know if the features are relevant to them before clicking the link to it.

Reply:
Suggest this line just to have the same title as the linked page:
"The 2D Renderer supports [URP Renderer Features](urp-renderer-features). The setup for the features are called before any of the 2D built-in passes are queued. Refer to the **URP Renderer Features** documentation for more information."

Reply:
Did what you suggested in the latest commit.